dstep/driver/Application.d

```diff
@@ -48,16 +46,22 @@ class Application : DStack.Application
 
     protected override void run ()
     {
+        derr.writefln(arguments.rawArgs);
         handleArguments;
 
         if (arguments.argument.input.hasValue)
         {
-            inputFiles ~= arguments.argument.input;
-            startConversion(inputFiles.first);
+            auto inputFiles = arguments.argument.input.values[1 .. $];
+            foreach(string fileName; inputFiles)
+            {
+                auto conversionTask = task!startParsingFile(language, argsToRestore, fileName, true, arguments, compiler, compilerArgs);
+                conversionTask.executeInNewThread();
+            }
+            startParsingFile(language, argsToRestore, arguments.argument.input.values[0], false, arguments, compiler, compilerArgs);
```

Comment:
I see. My original idea of using `parallel` would handle the thread scheduling automatically. Meaning a new task would not necessarily create a new thread. But that might be a good idea anyway.